examples/RingRodCase/ring_rod.py

```diff
@@ -0,0 +1,117 @@
+import numpy as np
+from elastica import *
+
```

Comment:
let's start clarifying namespaces hereon as a practice?
```suggestion
import elastica as ea

```
Then modify follow stuff accordingly.